data/2016/dataSources/ServerItemDefinitions.json

```diff
@@ -5599,7 +5599,7 @@
         "PASSIVE_ABILITY_ID": 0,
         "COST": 0,
         "ITEM_CLASS": 16053,
-        "MAX_STACK_SIZE": 9999,

```

Comment:
Thats just gonna lower the QOL, even if it worked like that back in the day

Reply:
Maybe having a max of 1 isn't great for QOL, but people are literally carrying around 10+ med kits, it makes bandages and other healing types completely useless. I don't think 1 is great either, maybe 3 or 4 max stack would be solid? People running around with 10+ is ridiculous though.

Reply:
Make it more rare maybe

Reply:
Changing stack size doesnt change how many a player can carry, for that you'd have to change the bulk per item. Having a higher stack size is just a QOL change for people who want to carry around more medkits.

Reply:
I see. I'll revert this change and yea it's kinda pointless b/c even with 1 stack size ppl can just carry more. Changing the bulk would be better (if we want to change it in the future :D)